kcidb/monitor/output.py

```diff
@@ -136,9 +138,24 @@ def render(self):
             ready to send, but missing the From header.
         """
         email = EmailMessage()
-        email["Subject"] = self.message.summary + self.obj.summarize()
+        subject = self.message.summary + self.obj.summarize()
+        email["Subject"] = subject
         email["To"] = ", ".join(self.message.recipients)
         email["X-KCIDB-Notification-ID"] = self.id
         email["X-KCIDB-Notification-Message-ID"] = self.message.id
-        email.set_content(self.message.description + self.obj.describe())
+        message = self.message.description + self.obj.describe()
+        email.set_content(message)
+        formatted_subject = html.escape(subject)
+        formatted_message = html.escape(message)
+        html_msg = (textwrap.dedent(f"""
+                        <html>
+                            <head>
+                                <title>{formatted_subject}</title>
+                            </head>
+                            <body>
+                                <pre>{formatted_message}</pre>
+                            </body>
+                        </html>
+                    """))
```

Comment:
Right, `textwrap.dedent()` wouldn't work like this, because it tries to "dedent" a text which has lines starting at the first column, because we've substituted the message, which has newlines in there with text starting at the first column.
So we gotta first "dedent" the format string, and then substitute our subject and message.

Could you switch this to first call `textwrap.dedent()` and then use the `format()` method on the resulting string?
Oh, and also, could you please remove the extra parentheses around the expression?